NitroxClient/Communication/Packets/Processors/PlayerKickedProcessor.cs

```diff
@@ -16,8 +16,15 @@ public UserKickedProcessor(IMultiplayerSession session)
 
         public override void Process(PlayerKicked packet)
         {
+            string reason = Language.main.Get("Nitrox_PlayerKicked");
```

Comment:
Minor: Rename variable to `message` instead of `reason` because it (looks like) the whole content of the kick dialog.